### PR TITLE
TDI-44553 - Fix failing SnowflakeWriterTest (locale dependent)

### PR DESCRIPTION
**Pull request by coheigea**
https://jira.talendforge.org/browse/TDI-44553

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/423/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/423/)

File: components/components-snowflake/components-snowflake-runtime/src/test/java/org/talend/components/snowflake/runtime/SnowflakeWriterTest.java [hunk 1]
```diff
@@ -142,6 +142,7 @@ public void testWrite() throws Exception {
         SimpleDateFormat timeFormatter = new SimpleDateFormat("HH:mm:ss.SSS");
         timeFormatter.setTimeZone(TimeZone.getTimeZone("GMT"));
         SimpleDateFormat timestampFormatter = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSSXXX");
+        timestampFormatter.setTimeZone(TimeZone.getTimeZone("GMT"));
```
Comment (mbasiuk-talend) [inline components/components-snowflake/components-snowflake-runtime/src/test/java/org/talend/components/snowflake/runtime/SnowflakeWriterTest.java:145]: Maybe we can define already existing formatters from util class:
```
        Formatter formatterClass = new Formatter();
        SimpleDateFormat dateFormatter = formatterClass.getDateFormatter();
        SimpleDateFormat timeFormatter = formatterClass.getTimeFormatter();
        SimpleDateFormat timestampFormatter = formatterClass.getTimestampFormatter();
```

Comment (coheigea) [reply components/components-snowflake/components-snowflake-runtime/src/test/java/org/talend/components/snowflake/runtime/SnowflakeWriterTest.java:145]: @mbasiuk-talend Which util class are you referring to?

Comment (mbasiuk-talend) [reply components/components-snowflake/components-snowflake-runtime/src/test/java/org/talend/components/snowflake/runtime/SnowflakeWriterTest.java:145]: https://github.com/Talend/components/blob/master/components/components-snowflake/components-snowflake-runtime/src/main/java/org/talend/components/snowflake/runtime/Formatter.java

Comment (coheigea) [reply components/components-snowflake/components-snowflake-runtime/src/test/java/org/talend/components/snowflake/runtime/SnowflakeWriterTest.java]: @mbasiuk-talend Yes that works for me, I've pushed an update - please take another look.

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/425/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/425/)

**Pull request merged**